### ISSUE TITLE
Make this into a Data Package

### ISSUE THREAD

**Issue by rgrp** [okfn/dataportals.org#64]
Add datapackage.json in data directory

**Comment by alexfriant** [comment]
This is great progress! I would be happy to populate the "descriptions". Is there a document anywhere with descriptions for our portals.csv columns?

**Comment by Stephen-Gates** [comment]
@alexfriant [This Google Sheet](http://goo.gl/PfHswi) is all I am aware of in the way of descriptions.

**Comment by rgrp** [comment]
@alexfriant that would be *brilliant* - I was just about to say we need the descriptions. Please start adding them in the datapackage.json and we can use and improve on the info from the sheet that @Stephen-Gates pointed to.

**Comment by Stephen-Gates** [comment]
Thanks for the descriptions @alexfriant #72. I've just noticed a number of type errors (not yours). A number of fields are defined as numbers instead of strings and could be defined more strongly eg. uri, date, etc.  Do you want to have a go tightening up the definitions?

**Comment by alexfriant** [comment]
Absolutely! I hadn't even given those a look so those will be next on my
to-do list.

**Comment by todrobbins** [comment]
I know it's be a while, but what's the status of this development @rufuspollock @Stephen-Gates @alexfriant?

**Comment by alexfriant** [comment]
I got swamped and haven't come back to it. Apologies! If it's something you
feel like taking on, go for it. If not, I would love to revisit and get
back in to this effort. Let me know what your interest is :-)

Alex

**Comment by todrobbins** [comment]
Thanks for the update @alexfriant. I'm happy to pick up where you left off. Do we mainly need to tighten the definitions and data types in the Data Package?

**Comment by alexfriant** [comment]
From what I remember, that was my plan. Thanks!

**Comment by Mikanebu** [comment]
@rufuspollock Updated `datapackage.json` with the latest spec and validated using DataHub CLI tool. Please review PR https://github.com/okfn/dataportals.org/pull/153
